mesonbuild/wrap/wrap.py

```diff
@@ -218,17 +218,16 @@ def get_file(self):
 
     def get_git(self):
         revno = self.wrap.get('revision')
-        if self.wrap.values.get('clone-recursive', '').lower() == 'true':
-            subprocess.check_call(['git', 'clone', '--recursive', self.wrap.get('url'),
-                                   self.directory], cwd=self.subdir_root)
-        else:
-            subprocess.check_call(['git', 'clone', self.wrap.get('url'),
-                                   self.directory], cwd=self.subdir_root)
+        subprocess.check_call(['git', 'clone', self.wrap.get('url'),
+                                self.directory], cwd=self.subdir_root)
```

Comment:
__[Flake8]__ [E127] continuation line over-indented for visual indent

<sub>posted by Sider</sub>